.buildkite/pipeline.trigger.integration.tests.sh

```diff
@@ -62,8 +61,6 @@ for package in $(find . -maxdepth 1 -mindepth 1 -type d) ; do
     echo "          provider: \"gcp\""
     echo "        artifact_paths:"
     echo "          - build/test-results/*.xml"
-    echo "          - build/elastic-stack-dump/stack/check-*/logs/*.log"
-    echo "          - build/elastic-stack-dump/stack/check-*/logs/fleet-server-internal/**/*"
```

Comment:
Following the approach of not uploading logs in these packages,  instead of fixing the path, I prefer to remove those·